process-document/src/main/kotlin/com/ritense/processdocument/autoconfigure/ProcessDocumentAutoConfigurationK.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.processdocument.autoconfigure
+
+import com.ritense.document.service.DocumentService
+import com.ritense.processdocument.domain.impl.delegate.DocumentDelegateImpl
+import com.ritense.processdocument.service.ProcessDocumentService
+import com.ritense.valtimo.contract.authentication.UserManagementService
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class ProcessDocumentAutoConfigurationK {
```

Comment:
Is the `K` in the classname intended?

Reply:
Yes, its so that this kotlin AutoConfiguration wont conflict with the existing Java AutoConfiguration.

Reply:
Wouldn't the real solution be to just give it a more descriptive name?

Reply:
Renamed